### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tbhagwat3**
@rido-min @ericmitt could you please review this?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
+This bug bash is focussed on the new IoT Hub features, SDKs, and tooling for the Plug and Play Summer Refresh 2020 release. There are two options during this bug bash. 
 
-Review the docs in the private staging environment [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=pr-en-us-114965)
+Option 1 is to follow Quickstarts in the Plug and Play documentation to build the samples provided there and run them against the IoT Explorer, a tool we're providing with this release. 
 
-The [DTDL v2 Spec](https://microsoft.sharepoint.com/:w:/t/Azure_IoT/IoTPlat/EaotICYXiQVOh_xnBggJwqMBMKGoOyMzV1qFYsOdcyvOVw?e=VyG0QN) can be used as a reference for the language. An easiest way to get started with the language is by looking at the [sample models](./models) created for this Bug Bash.
+Option 2 is to use a set of intructions we're providing to build a simulated Plug and Play device and a solution to validate the workflow end to end.
```
Comment (rido-min) [inline README.md:10]: I think Option 2 should also use the docs, we are providing Stefan's deck just as a workaround but ideally it should not be required and docs should be enough. 

**docs should provide instructions to build pnp solutions**

I would leave Option 2 for the second BugBash.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,24 @@
...
 
-### BugBash support and feedback
+The [DTDL v2 Spec](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md) can be used as a reference for the language. Use the [samples](https://github.com/Azure/opendigitaltwins-dtdl/tree/master/DTDL/v2/samples) we've provided to get started.
```
Comment (rido-min) [inline README.md:12]: we might want to include the short link https://aka.ms/dtdl

File: README.md [hunk 3]
```diff
@@ -1,29 +1,24 @@
...
-The required hub version is only  available in the canary environment, to create a new hub you can follow the instructions from the docs but using the **subscription** and **regions** described below:
-
-- Request access to the `IOTPNP_TEST_BY_MAIN` subscription, via the Teams channel
-- Create a new IoT Hub under this subscription, wihtin the resource group `BugBash` in one of the supported regions: `Central US EUAP` or `East US 2 EUAP`
+- If you have bugs or feature request, please use this [Bug Template](https://msazure.visualstudio.com/One/_workitems/create/Bug?templateId=221e542f-3428-49ba-951b-5ba1dce3f9a9&ownerId=1e65a829-00c0-4dc9-8088-d41678a0d033). You can query existing bugs in this [PnP BugBash Query](https://msazure.visualstudio.com/One/_queries/query-edit/07523176-81f2-4eb3-a795-8a483cd30310/)
 
+### Getting started, environment and tooling
+#### IoT Hub
+- The required hub version is only available in the canary environment. To get started request access to the `IOTPNP_TEST_BY_MAIN` subscription, via the Teams channel mentioned above. 
+- Create S1 IoTHub in region ‘Central US EUAP’.
```
Comment (rido-min) [inline README.md:21]: I've been using East US EUAP, just wanted to double check

File: README.md [hunk 4]
```diff
@@ -39,246 +34,16 @@ To create the hub from the portal make sure you select the right subscription, r
 
 > Note. Make sure you get the IoT Hub connection string to be able to configure IoT explorer.
 
-#### SDK support
```
Comment (rido-min) [inline README.md:42]: Why remove this section?

File: README.md [hunk 5]
```diff
@@ -39,246 +34,16 @@ To create the hub from the portal make sure you select the right subscription, r
...
-2. Install the private version of IoT Explorer
-3. Create a device registration in IoT explorer and get the device connection string.
-4. Use the connection string to connect the sample devices to hub
-5. IoT Explorer should show the `IoT Plug and Play components` tab
-6. Configure IoT Explorer to load the models from a local folder
-7. Inspect Telemetry, Update properties and invoke commands from IoT Explorer
-
->Note: These steps are valid to validate any PnP device
-
-### 2. Create the Thermostat Device simulator
```
Comment (rido-min) [inline README.md:220]: Can we include some requirements to create the device simulator? Can we provide sample models?

File: README.md [hunk 6]
```diff
@@ -39,246 +34,16 @@ To create the hub from the portal make sure you select the right subscription, r
...
-#### Things to try in the device app
-
-- Edit the model with the VSCode extension and verify how the systax is validated
-- Implement a device simulator in C, Python or node to match the model interfaces provided in [`Thermostat.json`](./models/Thermostat.json)
-- Combine telemetry messages with more than one property
-- Add a new command with complex types as input or output parameters
-- Update a property when the device is offline and verify it's applied when the device reconnects.
-- Evaluate error messages when the property names do not match the names in the model.
-
-### 3. Create the Cloud Application
```
Comment (rido-min) [inline README.md:244]: include requirements/ideas to implement a service side solution

File: README.md [hunk 7]
```diff
@@ -39,246 +34,16 @@ To create the hub from the portal make sure you select the right subscription, r
...
-- Share a model
-- Publish a model
-- Manage user access
-
-#### Things to try
-
-- Create your own AAD tenant by following the instructions [here](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-access-create-new-tenant) and add a user to that tenant. Login with the user in the model repo portal where you will see yourself as first tenant admin.
-- Try manage models and user access with your own tenant following the same steps in the howto-manamge-models article.
-
-### 5. Digital Twin Change Notification
```
Comment (rido-min) [inline README.md:275]: We need to include more details on how to subscribe to change notifications

File: README.md (hunk 1 above)
Comment (NSannala) [reply README.md:10]: Unless there is a downside to it, i will leave Option 2 given this is going to be continuous 2 week bug bash that folks can participate at their convenience.

**Pull request merged**